plugins/woocommerce-blocks/assets/js/blocks/collection-filters/block.json

```diff
@@ -4,26 +4,23 @@
 	"title": "Collection Filters",
 	"description": "A block that adds product filters to the product collection.",
 	"category": "woocommerce",
-	"keywords": [
-		"WooCommerce",
-		"Filters"
-	],
+	"keywords": [ "WooCommerce", "Filters" ],
 	"textdomain": "woocommerce",
 	"supports": {
 		"html": false,
 		"reusable": false
 	},
-	"usesContext": [
-		"query",
-		"queryId"
-	],
+	"usesContext": [ "query", "queryId" ],
 	"providesContext": {
 		"collectionData": "collectionData"
 	},
 	"attributes": {
 		"collectionData": {
 			"type": "object",
 			"default": {}
+		},
```

Comment:
some autoformatting left behind from editing this file. happy to remove this from PR if needed.